### PR TITLE
Refactor `repo` related functions

### PR DESCRIPTION
**Pull request by 4n12i**
Reflect changes in #191

### CODE REVIEW

File: src/cmd/arc.rs [hunk 1]
```diff
@@ -42,10 +42,10 @@ pub fn run(args: &Args) -> Result<()> {
 pub struct ArgsGuestKernelUprev {
     /// target cros repo dir
     #[argh(option)]
-    repo: Option<String>,
+    cros: Option<String>,
 }
 fn run_guest_kernel_uprev(args: &ArgsGuestKernelUprev) -> Result<()> {
-    let chroot = Chroot::new(&get_repo_dir(&args.repo)?)?;
+    let chroot = Chroot::new(&get_repo_dir(&args.cros)?)?;
```
Comment (mhiramat) [inline src/cmd/arc.rs:48]: Can you also rename `get_repo_dir` to `get_cros_dir`?

**Review by mhiramat — COMMENTED** [review]
Looks good to me except one item.

File: src/cmd/arc.rs (hunk 1 above)
Comment (4n12i) [reply src/cmd/arc.rs]: Sure, I fixed it!

**Review by mhiramat — APPROVED** [review]
This looks good to me.

**Review by brandonpollack23 — CHANGES_REQUESTED** [review]
I still have a problem with this name just being "cros" or just "arc"

It doesn't make sense and should probably be cros_repo and arc_repo

**Comment by 4n12i** [comment]
@brandonpollack23 @hikalium 
CC @mhiramat 

I have filed an issue https://github.com/google/lium/issues/218 on this topic, so could you discuss the option names there?  
Please merge this PR once and create another PR if you decide to change.

**Pull request merged**